### PR TITLE
SC-383: Only update R version

### PR DESCRIPTION
**Pull request by xschildw**
This PR rolls back the update of the OS and Rstudio but installs R 4.3 and associated libraries.
In progress, just want to see if basic R/RStudio works.

### CODE REVIEW

File: src/playbook.yaml [hunk 1]
```diff
@@ -65,19 +61,38 @@
           - libcurl4-openssl-dev
           - libapparmor1
           - libxml2-dev
-          - python3-boto3
+          - ibharfbuzz-dev
```
Comment (xschildw) [inline src/playbook.yaml:64]: Had to add these as I was re-installing the libraries

File: src/playbook.yaml [hunk 2]
```diff
@@ -65,19 +61,38 @@
...
+          - libpng-dev
+          - libtiff5-dev
+          - libjpeg-dev
+          - software-properties-common
 
-    # Install essential R packages
-    - name: Install synapser
-      # environment variable needed to communicate with the embedded python and install boto3 dependency
-      shell: "R -e \"Sys.setenv(SYNAPSE_PYTHON_CLIENT_EXTRAS='boto3'); install.packages('synapser', repos=c('http://ran.synapse.org', 'http://cran.fhcrc.org'), Ncpus = 2)\""
+    - name: Setup python 3.8
```
Comment (xschildw) [inline src/playbook.yaml]: 3.6.x installed. Need 3.8 for synapser.

File: src/playbook.yaml [hunk 3]
```diff
@@ -94,7 +113,7 @@
           DefaultLimitNOFILE=1000000
 
     - name: Download RStudio Server
-      get_url: url=https://download2.rstudio.org/server/jammy/amd64/rstudio-server-2023.03.1-446-amd64.deb dest=/tmp/rstudio.deb
+      get_url: url=https://download2.rstudio.org/server/bionic/amd64/rstudio-server-2022.02.0-443-amd64.deb dest=/tmp/rstudio.deb
```
Comment (xschildw) [inline src/playbook.yaml]: Note: we could get an newer version that still supports Ubuntu 18 (rstudio-server-2023.03.2-454-amd64.deb)

File: src/template.json [hunk 4]
```diff
@@ -63,7 +63,7 @@
     "InstanceType": "t3.small",
     "OwnerEmail": "it@sagebase.org",
     "SnapshotGroups": "all",
-    "SourceImage": "ami-0b710e028cd5ff838",
+    "SourceImage": "ami-0a3c3e0cb6ae0cc6c",
```
Comment (xschildw) [inline src/template.json]: packer-base-ubuntu-bionic-v1.0.10

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**